/users/stud09/martinsd/proj/buhayra/buhayra/insertPolygons.py:16: FutureWarning: The value of this property will change in version 1.0. Please see https://github.com/mapbox/rasterio/issues/86 for details.
  poly=tif2shapely(f)
/users/stud09/martinsd/local/miniconda2/envs/fiorio_orson/lib/python3.7/site-packages/rasterio/features.py:102: FutureWarning: GDAL-style transforms are deprecated and will not be supported in Rasterio 1.0.
  transform = guard_transform(transform)
Traceback (most recent call last):
  File "/users/stud09/martinsd/local/miniconda2/envs/fiorio_orson/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/users/stud09/martinsd/local/miniconda2/envs/fiorio_orson/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/users/stud09/martinsd/proj/buhayra/__main__.py", line 72, in <module>
    main()
  File "/users/stud09/martinsd/proj/buhayra/__main__.py", line 34, in main
    ipol.insertLoop()
  File "/users/stud09/martinsd/proj/buhayra/buhayra/insertPolygons.py", line 19, in insertLoop
    feat_id=insertNEB(feat)
  File "/users/stud09/martinsd/proj/buhayra/buhayra/insertPolygons.py", line 43, in insertNEB
    result = neb.update_one({'properties.id_jrc':feat["properties"]["id_jrc"] , 'properties.ingestion_time' :feat["properties"]["ingestion_time"] },{'$set':feat},upsert=True)
  File "/users/stud09/martinsd/local/miniconda2/envs/fiorio_orson/lib/python3.7/site-packages/pymongo/collection.py", line 995, in update_one
    session=session),
  File "/users/stud09/martinsd/local/miniconda2/envs/fiorio_orson/lib/python3.7/site-packages/pymongo/collection.py", line 851, in _update_retryable
    _update, session)
  File "/users/stud09/martinsd/local/miniconda2/envs/fiorio_orson/lib/python3.7/site-packages/pymongo/mongo_client.py", line 1247, in _retryable_write
    with self._tmp_session(session) as s:
  File "/users/stud09/martinsd/local/miniconda2/envs/fiorio_orson/lib/python3.7/contextlib.py", line 112, in __enter__
    return next(self.gen)
  File "/users/stud09/martinsd/local/miniconda2/envs/fiorio_orson/lib/python3.7/site-packages/pymongo/mongo_client.py", line 1576, in _tmp_session
    s = self._ensure_session(session)
  File "/users/stud09/martinsd/local/miniconda2/envs/fiorio_orson/lib/python3.7/site-packages/pymongo/mongo_client.py", line 1563, in _ensure_session
    return self.__start_session(True, causal_consistency=False)
  File "/users/stud09/martinsd/local/miniconda2/envs/fiorio_orson/lib/python3.7/site-packages/pymongo/mongo_client.py", line 1516, in __start_session
    server_session = self._get_server_session()
  File "/users/stud09/martinsd/local/miniconda2/envs/fiorio_orson/lib/python3.7/site-packages/pymongo/mongo_client.py", line 1549, in _get_server_session
    return self._topology.get_server_session()
  File "/users/stud09/martinsd/local/miniconda2/envs/fiorio_orson/lib/python3.7/site-packages/pymongo/topology.py", line 427, in get_server_session
    None)
  File "/users/stud09/martinsd/local/miniconda2/envs/fiorio_orson/lib/python3.7/site-packages/pymongo/topology.py", line 199, in _select_servers_loop
    self._error_message(selector))
pymongo.errors.ServerSelectionTimeoutError: 141.89.96.184:27017: timed out
